turn/src/server/mod.rs

```diff
@@ -92,6 +97,48 @@ impl Server {
         }
     }
 
+    /// Get information of [`Allocation`]s by specified [`FiveTuple`]s.
+    ///
+    /// If `five_tuples` is:
+    /// - [`None`]:               It returns information about the all
+    ///                           [`Allocation`]s.
+    /// - [`Some`] and not empty: It returns information about
+    ///                           the [`Allocation`]s associated with
+    ///                           the specified [`FiveTuples`].
+    /// - [`Some`], but empty:    It returns an empty [`HashMap`].
+    ///
+    /// [`Allocation`]: crate::allocation::Allocation
+    pub async fn get_allocations_info(
```

Comment:
Should this whole method be featured gated per the logic above?

Reply:
I do think we should. I've tried to explain why in previous answers above.